CHANGELOG.md

```diff
@@ -1 +1,15 @@
+<!-- markdownlint-disable MD024 -->
+
 # Change Log
+
+All notable changes to the "azurecontainerapps" extension will be documented in this file.
```

Comment:
IMO this line isn't required. If you'd rather keep it in though I would use the full extension name:

```suggestion
All notable changes to the "vscode-azurecontainerapps" extension will be documented in this file.
```